### PR TITLE
Create demos for R 

### PR DESCRIPTION
**Pull request by ohaanika**
## Demo Checklist

Creating the R equivalent of the [demos](https://github.com/plotly/dash-cytoscape) in the main directory of the repo:
- [ ] usage-advanced.py
- [ ] [WIP] usage-elements.py
- [x] usage-events.py
- [ ] usage-stylesheet.py

Creating the R equivalent of the [demos](https://github.com/plotly/dash-cytoscape/tree/master/demos) working correctly in Python:
- [x] usage-multiple-instances.py
- [x] usage-pie-style.py
- [x] usage-visual-style.py 
- [x] usage-grid-layout.py
- [x] usage-circle-layout.py

Creating the R equivalent of the [demons](https://github.com/plotly/dash-cytoscape/tree/master/demos) that are not fully functional yet in Python or are a WIP:
- [x] usage-animated-bfs.py
- [x] usage-breadthfirst-layout.py
- [x] usage-compound-nodes.py
- [ ] [WIP] usage-concentric-layout.py
- [ ] usage-concentric-social-network.py
- [ ] usage-cose-bilkent-layout.py
- [ ] [WIP] usage-cose-layout.py
- [ ] usage-dag-edges.py
- [ ] usage-edge-types.py
- [ ] usage-elements-extra.py
- [ ] usage-grid-social-network.py
- [ ] usage-initialisation.py
- [ ] usage-labels.py
- [ ] usage-linkout-example.py
- [ ] [WIP] usage-phylogeny.py
- [ ] usage-preset-animation.py
- [ ] usage-remove-selected-elements.py    
- [ ] usage-reset-button.py

## Pre-Merge checklist
- [ ] The project was correctly built with `npm run build:all`.
- [ ] If there was any conflict, it was solved correctly
- [ ] All changes were documented in CHANGELOG.md.
- [ ] All tests on CircleCI have passed.
- [ ] All Percy visual changes have been approved.
- [ ] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash Cytoscape core contributor.

## Reference Issues

Closes #[issue number]

## Other comments

### CODE REVIEW

**Comment by rpkyle** [comment]
I've removed the example which was unfinished (for now), and have edited out the very soon-to-be deprecated references to core component packages; I'll merge this into the v0.3.0 branch.

File: demosR/usage-compound-nodes.R [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# original demo: http://js.cytoscape.org/demos/compound-nodes/
+#
+# note: the Dash version is also uncentered, and otherwise it works
```
Comment (HammadTheOne) [inline demosR/usage-compound-nodes.R:3]:
```suggestion

```

File: demosR/usage-visual-style.R [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# original demo: http://js.cytoscape.org/demos/visual-style/
+
```
Comment (HammadTheOne) [inline demosR/usage-visual-style.R:2]:
```suggestion
# original demo: http://js.cytoscape.org/demos/visual-style/
#
# note: animation has not been implemented yet, please refer to code.
```

File: demosR/phylogeny/dummyData.R [hunk 3]
```diff
@@ -0,0 +1,248 @@
+dummyNodes <- list(
```
Comment (HammadTheOne) [inline demosR/phylogeny/dummyData.R:1]: Do the contents of `/demosR/phylogeny` serve any purpose?

I can't find any reason for them to be there, they don't affect the examples as far as I can tell. 

 :hocho:

Comment (rpkyle) [reply demosR/phylogeny/dummyData.R:1]: good catch -- I removed that demo (for now) because it was unfinished, will clean up this directory also.

**Review by HammadTheOne — APPROVED** [review]
:woman_dancing:

**Pull request merged**